org.eclipse.scanning.example/src/org/eclipse/scanning/example/scannable/MockNeXusScannable.java

```diff
@@ -19,6 +19,7 @@
 import org.eclipse.dawnsci.nexus.NexusException;
 import org.eclipse.dawnsci.nexus.NexusNodeFactory;
 import org.eclipse.dawnsci.nexus.NexusScanInfo;
+import org.eclipse.dawnsci.nexus.NexusScanInfo.NexusRole;
 import org.eclipse.dawnsci.nexus.NexusScanInfo.ScanRole;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.eclipse.dawnsci.nexus.NexusScanInfo.ScanRole'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)